BoothDownloader/src/Program.cs

```diff
@@ -179,6 +181,9 @@ private static async Task<int> Main(string[] args)
                 if (BoothHttpClientManager.IsAnonymous)
                 {
                     LoggerHelper.GlobalLogger.LogError("Cannot download Paid Items with invalid cookie.");
+                    LoggerHelper.GlobalLogger.LogInformation("Exiting in 5 seconds...");
```

Comment:
People can add booth URLs along with a collection command. Not sure if you want to exit here, code at the end will end the program anyways for nothing to download, maybe move this exit down there.

Reply:
when using the url handler if the token was invalid it would still get to this part of the code but make it impossible to read the output, maybe just thread sleep for 5 seconds instead?